client/extensions/woocommerce/style.scss

```diff
@@ -1,10 +1,57 @@
 .woocommerce {
 
+	.form-label {
+		margin-bottom: 16px;
+	}
+
 	.foldable-card.products__variation-card .foldable-card__content {
 		padding: 0;
 		border: 0;
 	}
 
+	.products__product-form-details {
+		.products__product-form-details-featured {
```

Comment:
When turned into pure CSS, this rule becomes
```css
.woocommerce .products__product-form-details .products__product-form-details-featured { }
```
The nesting under `products__product-form-details` is redundant.